examples/bsearch.inputs

```diff
@@ -0,0 +1,14 @@
+{
+    "note": [
```

Comment:
Can you move those "note" comments to the beginning of the bsearch.masm file? It doesn't work on the playground (yet) having "notes" in the input